app/src/main/java/ch/epfl/sdp/appart/glide/visitor/GlideBitmapLoader.java

```diff
@@ -57,7 +58,7 @@ public void visit(MockDatabaseService database) {
         BitmapTarget target = new BitmapTarget(bitmapFuture);
         Glide.with(context)
                 .asBitmap()
-                .load(Uri.parse(database.prependAndroidFilePath(imagePath)))
+                .load(Uri.parse(database.prependAndroidFilePath("panorama_test.jpg")))
```

Comment:
Ok, does this allows to have a picture loading in the tests too ? I know before it was failing due to the path with the user id etc. It's a good idea.

Reply:
Yes it was a small change to make sure an image is loaded without issues